tests/unittests/tests-core/tests-core-xfa.c

```diff
@@ -23,21 +23,21 @@ XFA_USE_CONST(xfatest_t, xfatest_use_const);
 /* Verifying that cross file array linking is correct by iterating over an external array */
 static void test_xfa_data(void)
 {
-    unsigned n = XFA_LEN(xfatest_t, xfatest);
+    unsigned n = xfatest_len();
```

Comment:
Just some bike shedding, but:
 ```suggestion
    unsigned n = xfa_len(xfatest);
```
would be more idiomatic

Reply:
true, but, `xfatest` is of unknown type, so `xfa_len(xfatest)` can't be a regular function. There's already `XFA_LEN(xfatest)` as macro. (which I was about to deprecate in favor of the accessor, but maybe I don't?)

Reply:
What's wrong with the macro then?
It's certainly easier for discoverability than some bespoke generated function.